README.md

```diff
@@ -646,3 +647,147 @@ Lays out the specified DAG, assigning the following properties:
 
 If *size* is specified, sets this zherebko layout's size to the specified two-element array of numbers [*width*, *height*] and returns this zherebko layout.
 If *size* is not specified, returns the current layout size, which defaults to [1, 1].
+
+
+### Arquint
+
+This treats nodes not as points (i.e. producing x & y coordinates) but as rectangles.
+Each node has a property *heightRatio* specifying its height in comparison to other nodes.
+The algorithm is based off a PR by L. Arquint.
```

Comment:
I would rephrase this a little. Saying it's based off a PR is a little inaccurate because it was submitted as a PR in full. First, I would link to or reference either a publication or further material on the algorithm potentially a link to your personal website. If you want to emphasize the fact that you also wrote the corresponding code, then I would say something like, "This implementation was contributed by the author."